app.py

```diff
@@ -20,7 +21,7 @@
 def allowed_file(filename):
     # Check for allowed file extension
     return '.' in filename and \
-           filename.rsplit('.', 1)[1] in ALLOWED_EXTENSIONS
+           filename.rsplit('.', 1)[1] in ALLOWED_EXTENSIONS or ALLOWED_EXTENSIONS_DESKTOP or ALLOWED_EXTENSIONS_LOGO
```

Comment:
nope. These are different file types

Reply:
Ok @tabesin, what are the changes that you are suggesting?

Reply:
Think about the logic. Each field has a permitted file type range. If you do it like that you can upload e.g. images by all uploaders because the condition is satisfied.

Reply:
@tabesin, No that will not happen, as we have already taken care of that in the html file using `accept` attribute in the input tag in index.html 
The image uploader will only take image files:
`<input type="file" name="myImage" id="file" class="file form-control" title="Default Wallpaper Image" 
onchange="$(this).prev('span').text($(this).val()!=''?$(this).val():'Browse')" `accept="image/*"` />`

The logo will only accept svg files:-
`<input type="file" name="myImage" id="file" class="file form-control" title="Default Desktop Logo" 
onchange="$(this).prev('span').text($(this).val()!=''?$(this).val():'Browse')" accept=".svg" />`
Same for zip files;

Reply:
@tabesin, No that will not happen, as we have already taken care of that in the html file using accept attribute in the input tag in index.html
The image uploader will only take image files:
<input type="file" name="myImage" id="file" class="file form-control" title="Default Wallpaper Image" onchange="$(this).prev('span').text($(this).val()!=''?$(this).val():'Browse')" accept="image/*" />

The logo will only accept svg files:-
<input type="file" name="myImage" id="file" class="file form-control" title="Default Desktop Logo" onchange="$(this).prev('span').text($(this).val()!=''?$(this).val():'Browse')" accept=".svg" />
Same for zip files;

Reply:
What is the purpose of allowed_file then?

Reply:
Please also mind that in the case of .tar.gz the file name extention has two points

Reply:
We need to have the accept attribute otherwise issue #261 will occur
I think think allowed_file to make flsdk know what extenaions to accept
I think. other contributors can also provide suggestions

Reply:
client side things can be played with but server side things cannot be

Reply:
@xeon-volt what do you think it should be done?